internal/painter/gl/draw.go

```diff
@@ -224,10 +224,13 @@ func roundToPixel(v float32, pixScale float32) float32 {
 }
 
 func roundToPixelCoords(size fyne.Size, pos fyne.Position, pixScale float32) (fyne.Size, fyne.Position) {
-	size.Width = roundToPixel(size.Width, pixScale)
-	size.Height = roundToPixel(size.Height, pixScale)
+	end := pos.Add(size)
+	end.X = roundToPixel(end.X, pixScale)
```

Comment:
A great fix thanks, but it looks like these two lines are duplicated - I don't think that is needed

Reply:
Its not duplicated, its performing the rounding to X and Y co-ordinates one by one.
Initially `pos`, and `size` are not yet rounded, so `end := pos.Add(size)` requries rounding off.

Reply:
Ah yes I see that now sorry!